comp/otelcol/collector-contrib/impl/collectorcontrib.go

```diff
@@ -176,6 +177,7 @@ func (c *collectorcontribImpl) OTelComponentFactories() (otelcol.Factories, erro
 	var errs error
 
 	connectorsList := []connector.Factory{
+		datadogconnector.NewFactory(),
```

Comment:
Can we first make connector use `https://github.com/DataDog/datadog-agent/tree/main/comp/otelcol/otlp/components/statsprocessor` and then import into Agent ?

Reply:
OK, we will wait for 0.102 release then

Reply:
Done